halodrops.cfg

```diff
@@ -0,0 +1,19 @@
+[MANDATORY]
+data_directory = ./dropsonde_data
+[OPTIONAL]
+path_to_flight_ids = {platform}
+path_to_l0_files = {platform}/{flight_id}/dropsondes/Level_0/
+
+[processor.Sonde.get_l2_filename]
+l2_filename_template = PERCUSION_{platform}_{launch_time}_{flight_id}_{serial_id}_Level_2.nc
+
+[processor.Gridded.get_l3_filename]
+l3_filename_template = PERCUSION_{platform}_{flight_id}_Level_3.nc
+
+
+
+
+
+#[processor.Sonde.interpolate_alt]
+#method = linear_interpolate
+

```

Comment:
I would clean the config script to _i)_ only include lines that are actually run, and _ii)_ don't have any blank lines